doc/AddASetting.md

```diff
@@ -1,40 +1,40 @@
 # Adding a Settings Property
 
-1. Add to wincon.w
-    * THIS IS NOT IN OPENCONSOLE. Make sure you update
-      `.../console/published/wincon.w` in the OS repo when you submit the PR.
+### 1. Add to wincon.w
+* THIS IS NOT IN OPENCONSOLE. Make sure you update
+      `.../console/published/wincon.w` in the Open Source repository when you submit the Pull Request.
       The branch won't build without it.
-    * For now, you can update winconp.h with your consumable changes.
-    * define registry name (ex `CONSOLE_REGISTRY_CURSORCOLOR`)
-    * add the setting to `CONSOLE_STATE_INFO`
-    * define the property key ID and the property key itself
-        - Yes, the large majority of the `DEFINE_PROPERTYKEY` defs are the same, it's only the last byte of the guid that changes
+* For now, you can update winconp.h with your consumable changes.
+* Define the registry name (e.g. `CONSOLE_REGISTRY_CURSORCOLOR`)
+* Add the setting to `CONSOLE_STATE_INFO`
+* Define the property key ID and the property key itself (Yes, the large majority of the `DEFINE_PROPERTYKEY` defs are the same, it's only the last byte of the guid that changes)
 
-2. Add matching fields to Settings.hpp
-    - add getters, setters, the whole drill.
+### 2. Add matching fields to Settings.hpp
+Add getters, setters, the whole drill.
 
-3. Add to the propsheet
-    - We need to add it to *reading and writing* the registry from the propsheet, and *reading* the link from the propsheet. Yes, that's weird, but the propsheet is smart enough to re-use ShortcutSerialization::s_SetLinkValues, but not smart enough to do the same with RegistrySerialization.
-    - `src/propsheet/registry.cpp`
-        -  `propsheet/registry.cpp@InitRegistryValues` should initialize the default value for the property.
-        -  `propsheet/registry.cpp@GetRegistryValues` should make sure to read the property from the registry
+### 3. Add to the propsheet
+We need to add it to *reading and writing* the registry from the propsheet, and *reading* the link from the propsheet. <br>
+Yes, that's weird, but the propsheet is smart enough to re-use ShortcutSerialization::s_SetLinkValues, but not smart enough to do the same with RegistrySerialization. <br>
+- `src/propsheet/registry.cpp`
+   -  `propsheet/registry.cpp@InitRegistryValues` should initialize the default value for the property.
+   -  `propsheet/registry.cpp@GetRegistryValues` should make sure to read the property from the registry
 
-4. Add the field to the propslib registry map
+### 4. Add the field to the propslib registry map

```

Comment:
Making these into h3's without any sub-points feels stylistically weird to me - @bitcrazed / @cinnamon-msft  want to weigh in on this?

Reply:
@zadjii-msft I agree Mike; I did find it to be *'stylistically weird'* as well, but then I thought it would be breaking the uniformity otherwise.

Reply:
I feel making them bold would be better than h3's